httpclient/src/test/java/feign/httpclient/ApacheHttpClientTest.java

```diff
@@ -188,6 +188,20 @@ public void postWithSpacesInPath() throws IOException, InterruptedException {
                 .hasBody("foo");
     }
 
+    @Test
```

Comment:
Clearly we want a test with the class effected! what happens if apache is deleted or moved out of the repo? the actual change occurred to RequestTemplate.

tests in http clients are often to ensure the client isn't interfering. For example, some http clients affect headers that are set. When making a change to feign, it is best to place the test where it is most relevant and in scope. I have no reason to believe that somehow apache is going to be a better place to test a change to RequestTemplate than RequestTemplateTest. If you somehow know better, please offer that reason.